pydantic/schema.py

```diff
@@ -643,6 +644,7 @@ def field_singleton_sub_fields_schema(
     (str, {'type': 'string'}),
     (SecretBytes, {'type': 'string', 'writeOnly': True}),
     (bytes, {'type': 'string', 'format': 'binary'}),
+    (StrictBool, {"type": "boolean"}),
```

Comment:
single quotes please.

Reply:
This doesn't fail the CI lint which makes it a bit difficult to contribute to. Especially since the default black formatter had a field day with the source code, and I had to unpick it.

Given that the black project prefers double quotes https://github.com/python/black/issues/118, why choose the option that is harder for contributors to use?

Reply:
I know all about black's decision on quotes and I strongly disagree. Unfortunately black's treatment of strings is so bad otherwise that just patching black to prefer single quotes doesn't work either.

There was a discussion [recently](https://github.com/samuelcolvin/pydantic/pull/572#discussion_r290389836) about linting for single quotes, it's something I want to do and haven't got around to yet.

We're definitely not changing the whole code base to double quotes.

Reply:
Not to argue, but solely for curiosity -- have you written the objections up somewhere? Would be interested :)

Reply:
No I haven't I read through the issue above and decided there was already too much noise.

put simply:
* `print(repr('foobar'))` givies `'foobar'`
* no shift required on most keyboards
* marginally less visual clutter
* because people wanted it and in the end, when there's no hard technical answer, that counts for a lot

Reply:
I think that final point is really strong. As is the visual clutter, because I've never noticed is so it's interesting that other people are affected by it, and it makes sense.

The 'shift' doesn't actually matter, because if you have black run on save, you can type single quotes anyway.

I think for me I'd rather go with the flow and make the contribution as easy as possible, both for the reviewer and the reviewee. Especially since I can imagine this isn't the first time you've had this conversation :joy_cat: 

Anyway, thanks for the info. Have changed to single quotes.

Reply:
You're quite right about the last point, I think the "visual clutter" point isn't really valid, I've just got used to single quotes and now prefer them.

There's another problem with black when `-S` is removed, https://github.com/python/black/issues/556. Which really would annoy me.

Basically I either need to build or find another tool which isn't trivial.